configs/peer/genesis.json

```diff
@@ -178,20 +178,70 @@
             "id": "ALICE_METADATA_ACCESS",
             "permissions": [
               {
-                "definition_id": "can_remove_key_value_in_user_account",
-                "params": {
-                  "account_id": {
-                    "AccountId": "alice@wonderland"
-                  }
-                }
+                "definition_id": "CanRemoveKeyValueInUserAccount",
+                "payload": [
```

Comment:
Maybe we can use some kind of loose format like `serde_json::Value` so it will be readable on one hand and can be parsed to strict structure if we needed?

Reply:
opened #3641